cardano-db-sync/src/Cardano/DbSync/Era/Util.hs

```diff
@@ -45,8 +45,8 @@ safeDecodeToJson tracer tracePrefix x = do
       pure Nothing
     Right json ->
       -- See https://github.com/IntersectMBO/cardano-db-sync/issues/297
-      if containsUnicodeNul json
+      if False && containsUnicodeNul json
```

Comment:
What's this?

Reply:
Good catch, I was testing if this check is redundant or not. It's not